lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -3205,6 +3199,8 @@ void DWARFASTParserClang::ParseChildParameters(
       break;
     }
   }
+
+  assert(function_param_names.size() == function_param_names.size());
```

Comment:
This looks like typo? Comparing with self.

Reply:
hah good catch! yea that's meant to say:
```
assert(function_param_types.size() == function_param_names.size());
```